pkg/stub/imagestreams.go

```diff
@@ -260,10 +260,6 @@ func (h *Handler) upsertImageStream(imagestreamInOperatorImage, imagestreamInClu
 }
 
 func (h *Handler) updateDockerPullSpec(oldies []string, imagestream *imagev1.ImageStream, opcfg *v1.Config) {
-	if len(opcfg.Spec.SamplesRegistry) == 0 {
-		return
-	}
-
```

Comment:
@gabemontero so if `SamplesRegistry` is cleared out, will we set the default registry to `registry.redhat.io`?

Reply:
set it back (if it had been changed from `registry.redhat.io`) yes @adambkaplan